docs/source/users/output/local_min.rst

```diff
@@ -4,4 +4,10 @@
 Local Minimiser Table
 #####################
 
-Place holder for the local minimiser table.
+The local min results show a ``True`` or ``False`` values together with a number. The ``True`` or ``False`` indicates whether the software finds a minimum with respect to the following criteria:
+
+    - :math:`||r|| \leq RES\_TOL`,
```

Comment:
These look odd in the math markdown, could RES_TOL and GRAD_TOL be \text{ … } or some other non italic style